### ISSUE TITLE
Shortcut support in FileExplorer

### ISSUE THREAD

**Issue by MikeKondr99** [tjx666/open-in-external-app#22]
When I am navigating in FileExplorer with keyboard and use shortcut "openInExternalApp.open" it's not working.

**Comment by tjx666** [comment]
How did you config it?

I have tried and no issue:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/41773861/152161932-34b8ff3b-cd03-4327-9e07-90bd365c8543.png">

**Comment by MikeKondr99** [comment]
I use "workbench.files.action.focusFilesExplorer" to navigate from Editor to FileExplorer then use arrows to select file and press shortcut.

This method works with split opening and I assumed will work with external opening.

I configured keybinding like this
![image](https://user-images.githubusercontent.com/97981268/152199881-1de6b59c-3334-4c2c-805f-b7f132841203.png)
(I tried another keybindings)

All work as intended when I select file with mouse and press keybinding.
